osu.Game/Scoring/ScoringValues.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using MessagePack;
+using osu.Game.Rulesets.Judgements;
+using osu.Game.Rulesets.Objects;
+using osu.Game.Rulesets.Scoring;
+
+namespace osu.Game.Scoring
+{
+    [MessagePackObject]
+    public struct ScoringValues
```

Comment:
Would like to see some xmldoc attached to this explaining what this class is and how it can be used. Something along the lines of "contains aggregated scoring data that fulfils the minimum requirements to calculate accuracy".